spec/diagnose_spec.rb

```diff
@@ -802,6 +843,38 @@
           },
           "system" => {}
         }
+      when :python
+        { "default" =>
+          { "ca_file_path" => ending_with("resources/cacert.pem"),
+            "diagnose_endpoint" => ending_with("diag"),
```

Comment:
Note that other integrations don't report the diagnose endpoint as a config option -- [there is an environment variable for it, but it's just read directly by the diagnose script](https://github.com/appsignal/appsignal-nodejs/blob/2dd8b61dfa9f34d706ac61f96d71b8d1087442e6/src/diagnose.ts#L281-L282), not added to the options/sources objects.

I'm not opposed to treating it as just another configuration option -- we already do for `endpoint`, after all. I'm only pointing out the difference in behaviour with the other integrations, in case it was not noticed/intentional.

Reply:
The Elixir integration also reports it. This one was highly inspired by it.